### ISSUE TITLE
mypy: How to fix FileNotFoundError: [Errno 2] No such file or directory: '.mypy_cache/3.7/config.data.json'

### ISSUE THREAD

**Issue by avatar-lavventura** [python/mypy#9977]
I am using `python-mypy` under `emacs v26.3`

I keep having following warning message, even I have the recent version of `python-mypy` not sure why this warning message is generated; is there any way to fix this:

```
Try installing a more recent version of python-mypy, and please open a bug report if the issue persists in the latest release.  Thanks!
Suspicious state from syntax checker python-mypy: Flycheck checker python-mypy returned 1, but its output contained no errors: Traceback (most recent call last):
  File "/home/alper/venv/bin/mypy", line 8, in <module>
    sys.exit(console_entry())
  File "/home/alper/venv/lib/python3.8/site-packages/mypy/__main__.py", line 11, in console_entry
    main(None, sys.stdout, sys.stderr)
  File "mypy/main.py", line 90, in main
  File "mypy/build.py", line 180, in build
  File "mypy/build.py", line 254, in _build
  File "mypy/build.py", line 2638, in dispatch
  File "mypy/build.py", line 2957, in process_graph
  File "mypy/build.py", line 3032, in process_fresh_modules
  File "mypy/build.py", line 1960, in load_tree
  File "mypy/metastore.py", line 98, in read
FileNotFoundError: [Errno 2] No such file or directory: '.mypy_cache/3.8/utils.data.json'

Try installing a more recent version of python-mypy, and please open a bug report if the issue persists in the latest release.  Thanks!

```

When ever I open `.emacs` this message pops-up in the mini-buffer. I keep seeing this error message in the mini-buffer and it is very annoying. Can I completely make it disable? Is it possible to completely ignore this message? 

----

my setup:

```
$ pip freeze | grep mypy
flake8-mypy==17.8.0
mypy==0.800
mypy-extensions==0.4.3
```

**Comment by hauntsaninja** [comment]
This is maybe more of an issue for whatever emacs plugin you're using to provide mypy.
